src/tests/work-item/work-item-list/work-item-detail.page.js

```diff
@@ -195,6 +195,11 @@ WorkItemDetailPage.prototype  = Object.create({}, {
     {
       return this.workItemDescriptionCancelIcon.click(); }
   },
+  titleValidation:   {
+    value: function ()
+    { 
+      return element(by.css(".clearfloat.alert.alert-danger")); }
```

Comment:
I think that we actually should have (2) separate checks for the alert. The current code is correct and works, but it does not verify the contents of the alert that is displayed. I'd suggest a 2-step approach:

```
  titleAlert:   {
    value: function ()
    { 
      return element(by.xpath(".//[@id='wi-title-div'][.//[contains(@class, 'alert-danger')]]")); }
  },


  titleAlertValidation:   {
    value: function ()
    { 
      return element(by.xpath(".//*[@id='wi-title-div']//p[.//text()[contains(.,'Title is required')]]")); }
  },

```

Other than this last suggested change - it looks great!